sale_order_type_automation/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Sale Order Type Automation
+==========================
+
+Add to kind of sales orders, options for when the sale is confirmed:
```

Comment:
Cambiar por:
Add to sale order type this options to be executed automatically on sale order confirmation:
    * Validate Invoice
    * Register and validate payment
    * Confirm picking and products transfer